docs/source/backends-qualcomm.md

```diff
@@ -51,7 +51,7 @@ You will need an Android smartphone with adb-connected running on one of below Q
  - SM8750 (Snapdragon 8 Elite)
  - SSG2115P
  - SSG2125P
- - SXR1230P
+ - SXR1230P (Linux Embedded)
```

Comment:
I thought SXR1230P is just a SoC similar to others. How is it special compared with the others?